test/WithdrawManager.test.js

```diff
@@ -0,0 +1,120 @@
+import chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import chaiBigNumber from 'chai-bignumber'
+import BigNumber from 'bignumber.js'
+
+import deployer from './helpers/deployer.js'
+import logDecoder from './helpers/log-decoder.js'
+import utils from 'ethereumjs-util'
+
+import {
+  getTxBytes,
+  getTxProof,
+  verifyTxProof,
+  getReceiptBytes,
+  getReceiptProof,
+  verifyReceiptProof
+} from './helpers/proofs'
+import { buildSubmitHeaderBlockPaylod, assertBigNumberEquality } from './helpers/utils.js'
+import { getBlockHeader } from './helpers/blocks'
+import MerkleTree from './helpers/merkle-tree'
+
+
+const rlp = utils.rlp
+const web3Child = new web3.constructor(
+  new web3.providers.HttpProvider('http://localhost:8546')
+)
+
+chai
+  .use(chaiAsPromised)
+  .use(chaiBigNumber(web3.BigNumber))
+  .should()
+
+contract('WithdrawManager', async function(accounts) {
+  let contracts, childContracts
+
+  beforeEach(async function() {
+    contracts = await deployer.freshDeploy()
+    childContracts = await deployer.initializeChildChain(accounts[0])
+  })
+
+  it('withdrawBurntTokens', async function() {
+    const user = accounts[0]
+    const amount = new BigNumber('10').pow(new BigNumber('18'))
+    await deposit(contracts.depositManager, childContracts.childChain, childContracts.rootERC20, user, amount)
+
+    const { receipt } = await childContracts.childToken.withdraw(amount)
+    const withdrawTx = await web3Child.eth.getTransaction(receipt.transactionHash)
+    const withdrawReceipt = await web3Child.eth.getTransactionReceipt(receipt.transactionHash)
+
+    const withdrawBlock = await web3Child.eth.getBlock(receipt.blockHash)
```

Comment:
[1/2] why not get all tx here with `web3Child.eth.getBlock(receipt.blockHash,true)` ?

Reply:
Awesome, didn't know about the boolean.!